Dev: Deploy Distributable Repos failed

Deploy Distributable Repos failed.

https://github.com/codeigniter4/CodeIgniter4/actions/runs/5972469494/job/16203057543
```
Run chmod +x ${GITHUB_WORKSPACE}/.github/scripts/validate-version
  chmod +x ${GITHUB_WORKSPACE}/.github/scripts/validate-version
  ${GITHUB_WORKSPACE}/.github/scripts/validate-version 4.4.0
  shell: /usr/bin/bash -e {0}
  env:
    LATEST_VERSION: 4.4.0
CodeIgniter version is not updated to v4.4.0
Error: Process completed with exit code 1.
```

References:
-  #6802

--- a/.github/scripts/validate-version
+++ b/.github/scripts/validate-version
@@ -10,7 +10,7 @@ FILES=("system/CodeIgniter.php" "user_guide_src/source/conf.py")
 LENGTH="${#FILES[@]}"
 
 for FILE in "${FILES[@]}"; do
-    COUNT="$((COUNT + $(grep -c "$FILE" -e "$1")))"
+    COUNT="$((COUNT + $(grep -c "$FILE" -e "'$1'")))"
 done
 
 if [[ $COUNT -ne $LENGTH ]]; then

